Cargo.toml

```diff
@@ -15,13 +15,17 @@ serde = { version = "1.0", features = ["derive"] }
 urlencoding = "1.0"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-reqwest = "0.10"
+isahc = { version = "0.9", features = ["http2", "text-decoding"], default_features = false }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 web-sys = { version = "0.3", features = ["RequestInit", "Headers", "Window", "Response", "console"] }
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
 
+[features]
+default = ["isahc-static-curl"]
+isahc-static-curl = ["isahc/static-curl"]
```

Comment:
This feature must not be available for wasm targets.

Reply:
Restricting features to certain targets is not (yet) possible in stable: https://github.com/rust-lang/cargo/issues/1197

Besides, without `isahc` being compiled, this feature won't do anything.